libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -311,22 +270,11 @@ class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
     fun showPrimaryRoute(style: Style) {
         jobControl.scope.launch(Dispatchers.Main) {
             mutex.withLock {
-                updateLayerVisibility(
-                    style,
-                    RouteLayerConstants.PRIMARY_ROUTE_TRAFFIC_LAYER_ID, Visibility.VISIBLE
-                )
-                updateLayerVisibility(
-                    style,
-                    RouteLayerConstants.PRIMARY_ROUTE_LAYER_ID, Visibility.VISIBLE
-                )
-                updateLayerVisibility(
-                    style,
-                    RouteLayerConstants.PRIMARY_ROUTE_CASING_LAYER_ID, Visibility.VISIBLE
-                )
-                updateLayerVisibility(
-                    style,
-                    RouteLayerConstants.RESTRICTED_ROAD_LAYER_ID, Visibility.VISIBLE
-                )
+                primaryRouteLineLayerGroup.ifEmpty {
+                    getLayerIdsForPrimaryRoute(style, sourceLayerMap).fold({ it }, { setOf() })
+                }.forEach {
```

Comment:
Let's change usage of this variable to a function instead, to avoid this construct being (inconsistently) copy-pasted.